hera_gpu/tests/test_redcal.py

```diff
@@ -0,0 +1,136 @@
+    def test_already_correct(self):
+        nubls = 15
+        nbls = NANT * (NANT - 1) // 2
+        for precision in (1,2):
+            gains = np.ones((NANT,NDATA), dtype=np.complex64)
+            ubls = np.ones((nubls,NDATA), dtype=np.complex64)
+            data = np.ones((nbls, NDATA), dtype=np.complex64)
+            wgts = np.ones((nbls, NDATA), dtype=np.float32)
+            conv_crit = 1e-4
+            maxiter = 100
+            check_every, check_after = 2, 1
+            ggu_indices = np.array([(i,j,i-j-1) for i in range(NANT) 
+                for j in range(i)], dtype=np.uint)
+            info = redcal.omnical(ggu_indices, gains, ubls, data, wgts,
+                        conv_crit, maxiter, check_every, check_after,
+                        precision=precision)
+            np.testing.assert_allclose(info['gains'], 1, 4)
+            np.testing.assert_allclose(info['ubls'], 1, 4)
+            np.testing.assert_allclose(info['chisq'], 0, 4)
+            np.testing.assert_allclose(info['iters'], 1, 4)
+            np.testing.assert_allclose(info['conv'], 0, 4)
+    def test_calibrate(self):
+        nubls = 15
+        nbls = NANT * (NANT - 1) // 2
+        for precision in (1,2):
+            gains = 1.1 * np.ones((NANT,NDATA), dtype=np.complex64)
+            ubls = np.ones((nubls,NDATA), dtype=np.complex64)
+            data = np.ones((nbls, NDATA), dtype=np.complex64)
+            wgts = np.ones((nbls, NDATA), dtype=np.float32)
+            conv_crit = 1e-3**precision
+            maxiter = 100
+            check_every, check_after = 2, 1
+            ggu_indices = np.array([(i,j,i-j-1) for i in range(NANT) 
+                for j in range(i)], dtype=np.uint)
+            info = redcal.omnical(ggu_indices, gains, ubls, data, wgts,
+                        conv_crit, maxiter, check_every, check_after,
+                        precision=precision)
+            np.testing.assert_allclose(info['gains'], 1, 4)
+            np.testing.assert_allclose(info['ubls'], 1, 4)
+            np.testing.assert_allclose(info['chisq'], 0, atol=10)
+            np.testing.assert_array_less(1, info['iters'])
+            np.testing.assert_array_less(info['conv'], conv_crit)
+
+class TestOmnicalSolver(unittest.TestCase):
+    def test_wrap(self):
+        NANTS = 18
+        antpos = linear_array(NANTS)
+        reds = om.get_reds(antpos, pols=['xx'], pol_mode='1pol')
+        info = redcal.RedundantCalibratorGPU(reds)
+        shape = (10, 10)
+        gains, true_vis, d = sim_red_data(reds, gain_scatter=.0099999, shape=shape)
+        w = dict([(k, 1.) for k in d.keys()])
+        sol0 = dict([(k, np.ones_like(v)) for k, v in gains.items()])
+        sol0.update(info.compute_ubls(d, sol0))
+        meta, sol = info.omnical_gpu(d, sol0, conv_crit=1e-12, gain=.5, maxiter=500, check_after=30, check_every=6)
+        #meta, sol = info.omnical(d, sol0, conv_crit=1e-12, gain=.5, maxiter=500, check_after=30, check_every=6)
```

Comment:
remove comment

Reply:
Would prefer to leave as a reminder of how to test vanilla omnical in place of omnical_gpu.

Reply:
I think it's okay to keep this around (though doesn't seem necessary because it takes the same arguments), but if we do keep it there should be an in-line comment explaining that it's the equivalent non-GPU call.